### PR TITLE
icons added

### PR DESCRIPTION
**Pull request by Cookman**
BEFORE CREATING THIS PR, have you bumped the package JSON where appropriate?

Ticket / Why this is needed:https://edozohq.atlassian.net/browse/OP-302

Link to Figma doc:

### CODE REVIEW

File: src/Icons/Icons.tsx [hunk 1]
```diff
@@ -92,6 +92,9 @@ import filtersUrl, { ReactComponent as Filters } from './icons/filters.svg';
 import standardArrowUrl, { ReactComponent as StandardArrow } from './icons/standard-arrow.svg';
 import bringToFrontUrl, { ReactComponent as BringToFront } from './icons/bring-to-front.svg';
 import sendToBackUrl, { ReactComponent as SendToBack } from './icons/send-to-back.svg';
+import alignLeftUrl, { ReactComponent as AlignLeft } from './icons/left-align.svg';
+import alignCenterUrl, { ReactComponent as AlignCenter } from './icons/center-align.svg';
+import alignrightUrl, { ReactComponent as Alignright } from './icons/right-align.svg';
```
Comment (martinmcneela) [inline src/Icons/Icons.tsx:97]:
```suggestion
import alignRightUrl, { ReactComponent as AlignRight } from './icons/right-align.svg';
```

File: src/Icons/Icons.tsx [hunk 2]
```diff
@@ -92,6 +92,9 @@ import filtersUrl, { ReactComponent as Filters } from './icons/filters.svg';
...
 
 const DEFAULT_ICON_COLOR = '#4c4c4c';
```
Comment (martinmcneela) [inline src/Icons/Icons.tsx:99]: You should be able to change this colour and all the icons in Storybook will change colour (with the exception of the app switcher icon). You can make sure this happens by removing the fill from all elements of the svg except the svg element itself.

**Review by martinmcneela — CHANGES_REQUESTED** [review]
The icons are not really aligned correctly, see the screenshot for an example. This is more than likely to do with how they were exported, happy to talk it through if you need too?
Screenshot 2022-08-17 at 09 48 20
Screenshot 2022-08-17 at 09 48 39

File: src/Icons/Icons.tsx (hunk 1 above)
Comment (Cookman) [reply src/Icons/Icons.tsx]: fixed

File: src/Icons/Icons.tsx (hunk 2 above)
Comment (Cookman) [reply src/Icons/Icons.tsx:99]: fixed

**Review by martinmcneela — APPROVED** [review]

**Pull request merged**